core/src/replay_stage.rs

```diff
@@ -396,11 +435,18 @@ impl ReplayStage {
             .lock()
             .unwrap()
             .reset(bank.last_blockhash(), bank.slot(), next_leader_slot);
-        debug!(
-            "{:?} voted and reset poh at {}. next leader slot {:?}",
+
+        let next_leader_msg = if let Some(next_leader_slot) = next_leader_slot {
+            format!("My next leader slot is #{}", next_leader_slot)
```

Comment:
we're gonna see this message on every vote, is that what you're going for?

Reply:
@rob-solana yeah, it's a bit repetitive but personally I found it helpful for being able to quickly tell when a node was going to be active. There are a few other messages that are printed every slot as well so it doesn't really add much noise